scripts/integration

```diff
@@ -49,9 +49,17 @@ if ! which shyaml &> /dev/null ; then
     exit 1
 fi
 
+GPG=gpg2
+if ! which "$GPG" ; then
+    GPG=gpg
+    if ! which "$GPG" ; then
+        echo "either gpg or gpg2 must be installed"
```

Comment:
If gpg or gpg2 is not found, should it exit with 1 ?

Reply:
Good catch. Will merge this bit with lines 62-65